media-sound/pianobar/pianobar-2024.12.21.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="A console-based client for the personalized online radio Pandora"
+DESCRIPTION="pianobar is a free/open-source, console-based client for the personalized online \
+radio Pandora (http://www.pandora.com).
+
+Features:
+
+- play and manage (create, add more music, delete, rename, ...) stations.
+- rate songs and explain why they have been selected.
+- upcoming songs/song history.
+- customize keybindings and text output.
+- remote control and eventcmd interface (send tracks to last.fm, for example).
+- proxy support for listeners outside the USA."
+HOMEPAGE="https://6xq.net/pianobar/"
+COPYRIGHT="2008-2014 Lars-Dominik Braun"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://6xq.net/pianobar/pianobar-$portVersion.tar.bz2"
+CHECKSUM_SHA256="16f4dd2d64da38690946a9670e59bc72a789cf6a323f792e159bb3a39cf4a7f5"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	pianobar$secondaryArchSuffix = $portVersion
+	cmd:pianobar = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libao$secondaryArchSuffix
+	lib:libavcodec$secondaryArchSuffix
+	lib:libavfilter$secondaryArchSuffix
+	lib:libavformat$secondaryArchSuffix
+	lib:libavutil$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libgcrypt$secondaryArchSuffix
+	lib:libgnutls$secondaryArchSuffix
+	lib:libjson_c$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libao$secondaryArchSuffix
+	devel:libavcodec$secondaryArchSuffix
+	devel:libavfilter$secondaryArchSuffix
+	devel:libavformat$secondaryArchSuffix
+	devel:libavutil$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libgcrypt$secondaryArchSuffix
+	devel:libgnutls$secondaryArchSuffix
+	devel:libjson_c$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+PATCH()
+{
+	sed -e "s|PREFIX:=/usr/local|PREFIX:=$prefix|" -i Makefile
+	sed -e "s|MANDIR:=\${PREFIX}/share/man|MANDIR:=$manDir|" -i Makefile
+	# in case we want to provide lib:libpiano/devel:libpiano at some point:
+	# sed -e "s|INCDIR:=\${PREFIX}/include|INCDIR:=$developDir/headers|" -i Makefile
```

Comment:
Doesn't this work without sed'n magic here? Like `make PREFIX=$prefix MANDIR=$manDir ...`

Reply:
Yep, just using `make PREFIX=$prefix MANDIR=$manDir install` in INSTALL() works fine without patching the Makefile.